appinfo/info.xml

```diff
@@ -31,9 +31,9 @@ The administrator is then logged-in as the user, to switch back to the regular u
     <bugs>https://github.com/nextcloud/impersonate/issues</bugs>
     <repository>https://github.com/nextcloud/impersonate.git</repository>
 
-    <screenshot>https://raw.githubusercontent.com/nextcloud/impersonate/master/screenshots/1.png</screenshot>
-    <screenshot>https://raw.githubusercontent.com/nextcloud/impersonate/master/screenshots/2.png</screenshot>
-    <screenshot>https://raw.githubusercontent.com/nextcloud/impersonate/master/screenshots/3.png</screenshot>
+    <screenshot>https://raw.githubusercontent.com/nextcloud/impersonate/stable14/screenshots/1.png</screenshot>
+    <screenshot>https://raw.githubusercontent.com/nextcloud/impersonate/stable14/screenshots/2.png</screenshot>
+    <screenshot>https://raw.githubusercontent.com/nextcloud/impersonate/stable14/screenshots/3.png</screenshot>
```

Comment:
We never did that for any of the apps. but well I guess it's fine.

Reply:
In a bookmarks release there was heavy UI changes in master and screenshots that have shown this. Way before a release, caused some confusion. Big changes are not expected here, it is the formally correct way though. No big feeling about that.